src/flows/sql/article_text_weekly_dedup.sql

```diff
@@ -0,0 +1,33 @@
+/*
+This will run after an hourly update only once on the weekend
+Will probably need to apply grants to the renamed live table
+Revoke grants will remove access to the old live table for better UX
+ */
+
+ use warehouse dpt_wh_3xl;
+    
+create or replace table article_content_ordered_new as (
+select RESOLVED_ID, 
+HTML, 
+TEXT, 
+SNOWFLAKE_LOADED_AT,
+TEXT_MD5,
+current_timestamp() as last_ordered
+from article_content_ordered_live
```

Comment:
@braunreyes what are your thoughts about merging the live and snapshot datasets here instead of doing the merge with a _UNION_ (view) at runtime as suggested in your [Implementation Decisions](https://github.com/Pocket/data-flows/pull/191#issue-1521252044)?

Reply:
I have tested this union with the query patterns that Evgeny is using where he is grabbing specific list of 10k rows at a time.  On the medium warehouse he is using it take an average of 30 seconds to return results.  Keeping the snapshot data separate allows me to only have to do a weekly reorder of the raw table.

Reply:
The raw table was initialized a few months back and is likely to grow. My thought was that merging snapshot and the raw live in the weekly process would avoid doing the dedupe/merge at runtime.

Do you see a specific disadvantage for materializing the weekly merge with the snapshot using the 3XL warehouse?